src/solvers/refinement/string_refinement_util.h

```diff
@@ -9,6 +9,8 @@
 #ifndef CPROVER_SOLVERS_REFINEMENT_STRING_REFINEMENT_UTIL_H
 #define CPROVER_SOLVERS_REFINEMENT_STRING_REFINEMENT_UTIL_H
 
+#include <memory>
+
 #include "string_builtin_function.h"
```

Comment:
The new include seems to be spurious?

Reply:
I'm afraid it isn't, this file uses `std::unique_ptr`, which previously worked as it was transitively included via message handler includes.